index.html

```diff
@@ -1105,6 +1105,57 @@
   This section has not been written.
 </aside>
 
+
+## Principles for Consent, Withdrawal of Consent, Opt-Outs, and Objections
+
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-consent-user-preference">
+    When any [=actor=] obtains [=consent=] for processing from a [=person=], the
+    actor should design the consent request so as to learn the person's true
+    intent to consent or not, and not to maximize the processing consented to.
+  </span>
+</p>
+</div>
+
+Attempts to obtain consent to [=processing=] that is not in accordance with the person's
+true preferences result in imposing unwanted [=privacy labour=] on the person, and may
+result in people erroneously giving consent that does not match their true preferences.
+
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-minimize-consent-requests">
+    An [=actor=] should avoid interrupting a [=person=]'s use of a site for
+	consent requests when an alternative is available.
```

Comment:
I want to be really concise:
```suggestion
    An [=actor=] should not interrupt a [=person=] to request their consent.
```

I can imagine "except in emergencies". I guess cookie banners are a case where no alternative is available, in which case "or when required by law"? This is still a "should", so sites can ignore it when there's a good reason, but I'd like to make it as firm as we plausibly can.